### PR TITLE
Add HPA up/downscale delay

### PR DESCRIPTION
**Pull request by discordianfish**
Still need to test these changes.

### CODE REVIEW

File: pkg/apis/kops/componentconfig.go [hunk 1]
```diff
@@ -313,8 +313,16 @@ type KubeControllerManagerConfig struct {
 	UseServiceAccountCredentials *bool `json:"useServiceAccountCredentials,omitempty" flag:"use-service-account-credentials"`
 	// HorizontalPodAutoscalerSyncPeriod is the amount of time between syncs
 	// During each period, the controller manager queries the resource utilization
-	// against the metrics specified in each HorizontalPodAutoscaler definition
+	// against the metrics specified in each HorizontalPodAutoscaler definition.
 	HorizontalPodAutoscalerSyncPeriod *metav1.Duration `json:"horizontalPodAutoscalerSyncPeriod,omitempty" flag:"horizontal-pod-autoscaler-sync-period"`
```
Comment (chrislovecnm) [inline pkg/apis/kops/componentconfig.go]: I dod not think we need these here https://kubernetes.io/docs/admin/kube-apiserver/  -  Can you check the docs on controller manager?  I think we just need the flags on that.

Comment (discordianfish) [reply pkg/apis/kops/componentconfig.go]: This is on the controller manager, or do I miss something?

Comment (chrislovecnm) [reply pkg/apis/kops/componentconfig.go]: KubeAPIServerConfig K c m has another struct

Comment (chrislovecnm) [reply pkg/apis/kops/componentconfig.go]: Or can I not read

Comment (chrislovecnm) [reply pkg/apis/kops/componentconfig.go]: Yup I cannot read. The diff got both me and @justinsb

**Pull request merged**